CHANGELOG.md

```diff
@@ -5,11 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.24.0] - 2022-07-22
+
+### Added
+
+- Adds `patch.estimates.create_air_shipping_estimate` method
```

Comment:
@holtbp it looks like it's `patch.estimates.createAirShippingEstimate` in node, no?

Reply:
You are correct